### PR TITLE
Task 6

### PR DESCRIPTION
**Pull request by TanyaLevchenya**

### CODE REVIEW

File: task_6/task_4/task1.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <title>Task1</title>
+</head>
+<body>
+  <main style="border: solid;">
+    
+  <div style="padding: 7px; 
+             margin: 25px; 
+             background-color: rgb(2, 143, 2); 
+             color: white;">
+       <span style="padding: 8px;">Div 1</span>
```
Comment (Killerkip10) [inline task_6/task_4/task1.html]: This `<span>` is redundant

File: task_6/task_4/task1.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  </div>
+    
+  <div style="padding: 7px; 
+             margin: 25px; 
+             background-color: rgb(8, 8, 204); 
+             color: white;">
+       <span style="padding: 8px;">Div 2</span>
```
Comment (Killerkip10) [inline task_6/task_4/task1.html]: Also

File: task_6/task_4/task2.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+           <span style="padding: 8px;">Div 1</span>
+      </div>
+  
+      <div style="height: 30px; 
+                 line-height: 30px; 
+                 background-color: rgb(7, 7, 214); 
+                 color: white; 
+                 width: 95.5%; 
+                 margin: -15px; 
+                 margin-left: 4.5%;">
```
Comment (Killerkip10) [inline task_6/task_4/task2.html]: Why did you use `%`. I think just `px` is enough.

File: task_6/task_4/task2.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+                 color: white;">
+           <span style="padding: 8px;">Div 1</span>
+      </div>
+  
+      <div style="height: 30px; 
+                 line-height: 30px; 
+                 background-color: rgb(7, 7, 214); 
+                 color: white; 
+                 width: 95.5%; 
+                 margin: -15px;
```
Comment (Killerkip10) [inline task_6/task_4/task2.html]: You should use negative margin only for `margin-top`

File: task_6/task_4/task2.html [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task2</title>
+</head>
+<body>
+  <main style="border: solid; border-width: 2px; padding: 18px 18px 33px">
+    
+      <div style="height: 30px;
```
Comment (Killerkip10) [inline task_6/task_4/task2.html]: Use `padding` instead of `height`

File: task_6/task_4/task2.html [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task2</title>
+</head>
+<body>
+  <main style="border: solid; border-width: 2px; padding: 18px 18px 33px">
+    
+      <div style="height: 30px; 
+                 line-height: 30px; 
+                 background-color: rgb(10, 148, 10);
```
Comment (Killerkip10) [inline task_6/task_4/task2.html]: Please use hex format for color

File: TrainingsSite/index.html [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+}
+
+
+
+</style>
+</head>
+
+<body style="margin: 0; 
+            background-color: #f0f2f3; 
+            height: 100vh;
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: Hmmm, what will happen if height will more than you window height?
Fix it. 
P.S. just remove it

**Review by Killerkip10 — COMMENTED** [review]
## Homework 4
### Task 1
15 points
### Task 2
20 points
### Task 3
15 points
### Task 4
- Layout mismatch. Header is not black (-5 points);
- 4.2 Background image occupy more than window height (-5 points);

40 points

**Total 90 points**

## Homework 5
### Task 1
- HTML Validator errors (-5 points);

20 points
### Task 2
25 points
### Task 3
- 3.5 Column with buttons should have static width (-5 points);
- Layout mismatch. 'Записаться' button has other background-color (-5 points);
 
40 points

**Total 85 points**

Resolve all conversations.
**Deadline 04.08.2020 00:00 else -20 points**

File: TrainingsSite/index.html [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+    </p>
+    <h2>demanded programs</h2>
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: Why your homework 7 is here?

**Comment by Killerkip10** [comment]
task_6/task_4/pexels-photo-583846.webp 
task_6/task_5/pexels-photo-583846 (1).webp 
Remove

File: TrainingsSite/index.html [hunk 9]
```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task7</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" type="text/css" href="./style.css">
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: It is task 7. 
Please remove it

File: TrainingsSite/index.html [hunk 10]
```diff
@@ -0,0 +1,195 @@
...
+<style>
+
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: Please align all css code and remove empty rows

File: TrainingsSite/index.html [hunk 11]
```diff
@@ -0,0 +1,195 @@
...
+      
+       
+        
+    }
+
+    h2 { text-transform: uppercase; 
+         text-align: center;
+
+    }
+    .text-button { color: rgb(75, 133, 241);
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: Please use hex format for color

File: TrainingsSite/index.html [hunk 12]
```diff
@@ -0,0 +1,195 @@
...
+    }
+    .text-button { color: rgb(75, 133, 241); 
+                   padding: 4px; 
+                   cursor: pointer;
+    }
+    .text-button:hover { color: rgb(16, 27, 90);
+
+    }
+    .button {
+            border-style: none;
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: Useless

File: TrainingsSite/index.html [hunk 13]
```diff
@@ -0,0 +1,195 @@
...
+            background-color: rgb(129, 241, 24);
+            color: white; 
+            border-radius: 4px;
+            text-align: center;
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: Will be better use for example:` padding: 7px 20px;`

File: TrainingsSite/index.html [hunk 14]
```diff
@@ -0,0 +1,195 @@
...
+            cursor: pointer;
+    }
+    .button:hover {
+        background-color: #43ca0d;
+    }
+    .button:active {
+        background-color: #32970a;
+    }
+
+    .text{  border-style: solid;
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: Class names are not informative

File: TrainingsSite/index.html [hunk 15]
```diff
@@ -0,0 +1,195 @@
...
+    }
+    .button:hover {
+        background-color: #43ca0d;
+    }
+    .button:active {
+        background-color: #32970a;
+    }
+
+    .text{  border-style: solid; 
+            border-width: 1px;
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: useless

File: task_6/task_4/task1.html (hunk 1 above)
Comment (TanyaLevchenya) [reply task_6/task_4/task1.html]: исправила

File: TrainingsSite/index.html [hunk 16]
```diff
@@ -0,0 +1,195 @@
...
+</head>
+
+<body style="margin: 0; 
+            background-color: #f0f2f3; 
+            height: 100vh; 
+            font-family: Roboto, serif;">
+
+<header>Trainings<sup>&#174;</sup>
+
+    <span style="padding-left: 35px;"><a href="./table1.html">List</a>
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: This `<span>` is redundant. Remove it

File: task_6/task_4/task1.html (hunk 2 above)
Comment (TanyaLevchenya) [reply task_6/task_4/task1.html]: исправлена

File: TrainingsSite/index.html [hunk 17]
```diff
@@ -0,0 +1,195 @@
...
+    <a href="./table2.html">Blog</a>
+    <a href="./table1.html">FAQ</a></span>
+    
+</header>
+
+<section style="width: 100%;
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: Sections already has `width: 100%` by default

File: TrainingsSite/index.html [hunk 18]
```diff
@@ -0,0 +1,195 @@
...
+               height: 100vh;
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: I think, you forgot subtract header height

File: TrainingsSite/index.html [hunk 19]
```diff
@@ -0,0 +1,195 @@
...
+               background-size: cover;">
+ 
+    <h3 style="color: white;
+              font-size: 55px;
+              padding-left: 150px;
+              margin: 0;">Start your<br>career</h3>
+
+</section>
+
+<main style="background-attachment: scroll;
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: background-attachment: scroll; is useless

File: TrainingsSite/index.html [hunk 20]
```diff
@@ -0,0 +1,195 @@
...
+ 
+    <h3 style="color: white;
+              font-size: 55px;
+              padding-left: 150px;
+              margin: 0;">Start your<br>career</h3>
+
+</section>
+
+<main style="background-attachment: scroll; 
+            padding: 0 20px 20px 20px;">
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: You can short it
padding: 0 20px 20px

File: TrainingsSite/index.html [hunk 21]
```diff
@@ -0,0 +1,195 @@
...
+    <div style="clear: both;"></div>
+   </div> 
+    <h2>upcoming courses</h2> 
+    <hr>
+    
+<table style="border-collapse: collapse;
+             border-radius: 5px;
+             background-color: white; 
+             width: 100%; 
+             min-height: 300px;">
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: Useless

File: TrainingsSite/index.html [hunk 22]
```diff
@@ -0,0 +1,195 @@
...
+    <tr style="border-bottom: solid rgba(83, 82, 82, 0.788) 1px">
+        <td style="padding-left: 15px; width: 41%;">
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: All columns stretch to all width automatic. Remove width

File: TrainingsSite/index.html [hunk 23]
```diff
@@ -0,0 +1,195 @@
...
+            <div class="text-button">Программирование на Go</div>
+            <div style="padding: 4px;">20.08.2020-20.12.2020</div>
+            <div style="font-size: 12px; padding: 4px;">Занятия вторник и четверг с 18.30 до 20.30</div> 
+        </td>
+    
+        <td>
+            <div style="padding: 4px;"><strong>755 BYN</strong></div>
+            <div style="font-size: 15px; padding: 4px;">128 уч.ч (три месяца)</div>
+            <div style="font-size: 12px; padding: 4px;"><strong>Осталось</strong> 10 из 20 мест</div>
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: Please, use css property instead of `<strong>`

File: TrainingsSite/index.html [hunk 24]
```diff
@@ -0,0 +1,195 @@
...
+        </td>
+    
+        <td style="table-layout: fixed; width: 18%; padding-right: 15px;">
```
Comment (Killerkip10) [inline TrainingsSite/index.html]: Please, use static value

File: task_6/task_5/table2.html [hunk 25]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Table2</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=sans-serif">
```
Comment (Killerkip10) [inline task_6/task_5/table2.html]: sans-serif is not a font. Remove it please.

File: task_6/task_4/task2.html (hunk 6 above)
Comment (TanyaLevchenya) [reply task_6/task_4/task2.html]: исправлено

File: TrainingsSite/index.html (hunk 7 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: task_6/task_5/table2.html (hunk 25 above)
Comment (TanyaLevchenya) [reply task_6/task_5/table2.html]: исправлено

File: TrainingsSite/index.html (hunk 10 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: task_6/task_4/task2.html (hunk 4 above)
Comment (TanyaLevchenya) [reply task_6/task_4/task2.html]: исправлено

File: task_6/task_4/task2.html (hunk 3 above)
Comment (TanyaLevchenya) [reply task_6/task_4/task2.html]: исправлено

File: task_6/task_4/task2.html (hunk 5 above)
Comment (TanyaLevchenya) [reply task_6/task_4/task2.html]: исправлено

File: TrainingsSite/index.html (hunk 11 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: TrainingsSite/index.html (hunk 12 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: TrainingsSite/index.html (hunk 21 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: TrainingsSite/index.html (hunk 22 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: TrainingsSite/index.html (hunk 23 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: TrainingsSite/index.html (hunk 24 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: TrainingsSite/index.html (hunk 8 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: ok

File: TrainingsSite/index.html (hunk 9 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: ok

File: TrainingsSite/index.html (hunk 14 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: TrainingsSite/index.html (hunk 15 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: TrainingsSite/index.html (hunk 16 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: TrainingsSite/index.html (hunk 17 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: TrainingsSite/index.html (hunk 19 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: TrainingsSite/index.html (hunk 20 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: TrainingsSite/index.html (hunk 18 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

File: TrainingsSite/index.html (hunk 13 above)
Comment (TanyaLevchenya) [reply TrainingsSite/index.html]: исправлено

**Pull request merged**